pkg/db/db_local/running_info.go

```diff
@@ -39,6 +39,10 @@ func newRunningDBInstanceInfo(cmd *exec.Cmd, listenAddresses []string, port int,
 		StructVersion:   RunningDBStructVersion,
 	}
 
+	if len(dbState.ListenAddresses) == 1 && dbState.ListenAddresses[0] == "*" {
+		addrs, _ := utils.LocalAddresses()
+		dbState.ListenAddresses = append(dbState.ListenAddresses, addrs...)
```

Comment:
Taken from https://github.com/turbot/steampipe/pull/3200/files#diff-5cdc872d3225f3c40af70a48a8baf0c3cc5757339aaad866130f0b342e83bc00L44

Shouldn't the first value be dropped instead?